app/src/main/java/co/nimblehq/compose/ecommerce/ui/bottomnavigationbar/Navigation.kt

```diff
@@ -9,6 +10,7 @@ import co.nimblehq.compose.ecommerce.ui.screen.home.HomeScreen
 import co.nimblehq.compose.ecommerce.ui.screen.product.ProductScreen
 import co.nimblehq.compose.ecommerce.ui.screen.search.SearchScreen
 
+@OptIn(ExperimentalMaterialApi::class)
```

Comment:
How about only

```suggestion
@ExperimentalMaterialApi
```

https://zsmb.co/opt-in-annotations/

Reply:
@luongvo If we use `@ExperimentalMaterialApi` we have to add the same annotation on the classes which use the `Navigation` as well, that's why I decided to use `@OptIn` instead.